### PR TITLE
ETL descriptions too long

### PR DESCRIPTION
**Pull request by bencoman**
Such a long test is really a comment, not a description to be used as a test identifier.
Make it so, per #1473

### CODE REVIEW

File: exercises/etl/canonical-data.json [hunk 1]
```diff
@@ -10,7 +10,10 @@
         "such that keys are integers. e.g. in JavaScript, it might look ",
         "like `transform( { 1: ['A'] } );`"
       ],
-      "description": "transforms the a set of scrabble data previously indexed by the tile score to a set of data indexed by the tile letter",
+      "description": "Transform",
```
Comment (SleeplessByte) [inline exercises/etl/canonical-data.json]: Consider 
```suggestion
      "description": "Transform score based index to letter based index",
```

Comment (bencoman) [reply exercises/etl/canonical-data.json]: Okay, a single word is a bit too minimal a description, but your suggestion for an identifier leaves my mind twisted and feeling its more a comment than a descriptor.  I committed an alternative that hopefully meets you in the middle.

File: exercises/etl/canonical-data.json [hunk 2]
```diff
@@ -10,7 +10,10 @@
         "such that keys are integers. e.g. in JavaScript, it might look ",
         "like `transform( { 1: ['A'] } );`"
       ],
-      "description": "transforms the a set of scrabble data previously indexed by the tile score to a set of data indexed by the tile letter",
+      "description": "Transform legacy to new",
+      "comments": [ "Transforms a set of legacy scrabble data stored as letters per score",
```
Comment (ErikSchierboom) [inline exercises/etl/canonical-data.json]: There are now two comments fields on the same level.

Comment (SleeplessByte) [reply exercises/etl/canonical-data.json]: Oh woops. Missed that in the diff.

**Comment by bencoman** [comment]
Lets hold further discussion here pending https://github.com/exercism/problem-specifications/pull/1525 and https://github.com/exercism/problem-specifications/issues/1473.

Note, I've just discovered I accidentally committed direct to master and I'll need to clean that up once a path forward is clear.  I was unfamiliar with using Github's web-editor and must have committed it directly before a second commit to a branch to issue this PR.

**Comment by SaschaMann** [comment]
> Note, I've just discovered I accidentally committed direct to master and I'll need to clean that up once a path forward is clear. I was unfamiliar with using Github's web-editor and must have committed it directly before a second commit to a branch to issue this PR.

That happens fairly easily with the web editor. Might be a good idea to setup branch protection for master on this repo to prevent accidental pushes without PRs.

**Comment by rpottsoh** [comment]
Errant commit has been reverted.

File: exercises/etl/canonical-data.json (hunk 2 above)
Comment (ErikSchierboom) [reply exercises/etl/canonical-data.json]: @bencoman The new comment is great. Could you perhaps move it to the existing comment on the same level?

**Comment by bencoman** [comment]
> move [new comment] to the existing comment on the same level?

done

File: exercises/etl/canonical-data.json [hunk 3]
```diff
@@ -1,19 +1,22 @@
 {
   "exercise": "etl",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "cases": [
     {
       "comments": [
+        "Transforms a set of legacy scrabble data stored as letters per score",
+        "to a set of data stored score per letter."
```
Comment (ErikSchierboom) [inline exercises/etl/canonical-data.json]: I think a comma is missing here.

```suggestion
        "to a set of data stored score per letter.",
```

**Comment by bencoman** [comment]
In future I'll use https://jsonlint.com/ to validate JSON before committing.

**Comment by rpottsoh** [comment]
https://jsonschemalint.com is also useful for checking your work against the [schema](https://github.com/exercism/problem-specifications/blob/master/canonical-schema.json).

**Pull request merged**